userspace/libscap/engine/gvisor/gvisor.h

```diff
@@ -102,6 +101,12 @@ class engine {
     int32_t process_message_from_fd(int fd);
     void free_sandbox_buffers();
 
+    std::vector<std::string> runsc(char *argv[]);
+    std::vector<std::string> runsc_list();
+    void runsc_trace_create(const std::string &sandbox_id, bool force);
+    void runsc_trace_delete(const std::string &session_name, const std::string &sandbox_id);
+    std::string generate_trace_session_config();
```

Comment:
I think this function name could be a little better or at least documented. By looking at it it looks like it's generating a configuration string, while it's actually writing a temp file.